### PR TITLE
Miscellaneous changes to server deployment settings

### PR DESCRIPTION
**Pull request by daisieh**
A bunch of tweaks to settings and whatnot, based on what we worked on in candig-dev deployment.

### CODE REVIEW

File: etc/env/example.env [hunk 1]
```diff
@@ -301,18 +303,18 @@ VAULT_SERVICE_PORT=8200
 VAULT_SERVICE_HOST=0.0.0.0
 #TODO: consolidate vault public and private domains
 VAULT_SERVICE_PUBLIC_URL=http://${VAULT_SERVICE_HOST}:${VAULT_SERVICE_PORT}
-VAULT_SERVICE_URL=http://candigv2_vault_1:${VAULT_SERVICE_PORT}
+VAULT_SERVICE_URL=http://candigv2_vault_1:8200
```
Comment (jimmyhli) [inline etc/env/example.env]: should underscore be avoided? or its ok

File: lib/candig-data-portal/docker-compose.yml [hunk 2]
```diff
@@ -27,6 +26,12 @@ services:
         - "traefik.http.routers.candig-data-portal.rule=Host(`candig-data-portal.${CANDIG_DOMAIN}`)"
         - "traefik.http.services.candig-data-portal.loadbalancer.server.port=${CANDIG_DATA_PORTAL_PORT}"
     logging: *default-logging
+    environment:
+      - REACT_APP_KATSU_API_SERVER=${TYK_KATSU_API_TARGET}
+      - REACT_APP_CANDIG_SERVER=${TYK_CANDIG_API_TARGET}
+        #- REACT_APP_FEDERATION_API_SERVER=http://${CANDIG_DOMAIN}:${FEDERATION_PORT}
+        #- REACT_APP_BASE_NAME='/'
+      - REACT_APP_SITE_LOCATION=UHN
```
Comment (jimmyhli) [inline lib/candig-data-portal/docker-compose.yml]: should this be a variable instead?

File: etc/env/example.env [hunk 3]
```diff
@@ -261,28 +263,28 @@ TYK_AUTH_API_SLUG=authentication
...
 TYK_KATSU_API_ID=31
 TYK_KATSU_API_NAME=Katsu
 TYK_KATSU_API_SLUG=katsu
-TYK_KATSU_API_TARGET=http://${CANDIG_DOMAIN}:${CHORD_METADATA_PORT}
+TYK_KATSU_API_TARGET=${CHORD_METADATA_PRIVATE_URL}
 TYK_KATSU_API_LISTEN_PATH=katsu
 
 ## api - mcode - candig-data-portal
 TYK_CANDIG_DATA_PORTAL_API_ID=41
 TYK_CANDIG_DATA_PORTAL_API_NAME="CanDIG Data Portal"
```
Comment (jimmyhli) [inline etc/env/example.env]: i think we changed this to `candig-data-portal`

Comment (jimmyhli) [reply etc/env/example.env]: as per discussions earlier lets use the same value for name and slug

also, i think we need to update our `policies.json` or its `tpl` to reflect the changes

File: lib/candig-data-portal/docker-compose.yml (hunk 2 above)
Comment (daisieh) [reply lib/candig-data-portal/docker-compose.yml]: What should its default value be? I just grabbed this change from the candig-dev deployment...

File: etc/env/example.env (hunk 3 above)
Comment (daisieh) [reply etc/env/example.env]: done

File: etc/env/example.env (hunk 1 above)
Comment (daisieh) [reply etc/env/example.env]: done

File: lib/candig-data-portal/docker-compose.yml (hunk 2 above)
Comment (daisieh) [reply lib/candig-data-portal/docker-compose.yml]: I made a new variable, CANDIG_SITE_LOCATION, in example.env.

Comment (jimmyhli) [reply lib/candig-data-portal/docker-compose.yml]: default can be `UHN`

**Review by jimmyhli — APPROVED** [review]

**Review by shaikh-rashid — APPROVED** [review]

**Pull request merged**